src/components/PageHero.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+
+class PageHero extends React.Component {
+    render() {
+        const {title, image, subTitle} = this.props
+        return (
+            <div className="page-hero">
+                <img
+                    src={ image } 
+                    alt={ title } 
+                    className="page-hero_img"
+                />
+                {console.log(image)}
```

Comment:
This is a beaut. Really like your use of the object deconstruction above 👏 . only thing here is to remove the console log and you'll be golden.